extension.html

```diff
@@ -0,0 +1,119 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset='utf-8'>
+    <title>PointerEvents - Level 2 - Extensions</title>
+    <link rel="stylesheet" type="text/css" href="https://w3c.github.io/uievents/spec-WD.css">
+    <script src='https://www.w3.org/Tools/respec/respec-w3c-common.js' class='remove'></script>
+    <script class='remove'>
+      var respecConfig = {
+        specStatus: "CG-DRAFT",
+        edDraftURI: "https://w3c.github.io/pointerevents/extensions.html",
+        editors: [{
+          name:        "Navid Zolghadr",
+          url:         "https://github.com/NavidZ",
+          company:     "Google",
+          companyURL:  "https://www.google.com/"
+        }],
+        shortName: "PointerEvent Level 2 Extensions",
+        otherLinks: [{
+          key: 'Participation',
+          data: [
+              {
+                value: 'GitHub repository',
+                href: 'https://github.com/w3c/pointerevents/'
+              } ,{
+                value: 'File a bug / view open issues',
+                href: 'https://github.com/w3c/pointerevents/issues'
+              }
+            ]
+          }],
+      };
+    </script>
+  </head>
+  <body>
+    <section id='abstract'>
+      <p>
+The features in this specification extend or modify those found in <a href="https://w3c.github.io/pointerevents/index.html">Pointer Events Level 2</a> that describes events and related interfaces for handling hardware agnostic pointer input from devices including a mouse, pen, touchscreen, etc.
+      </p>
+    </section>
+
+    <section id='sotd'>
+      <p>
+        This proposal was first introduced in this <a href="https://discourse.wicg.io/t/new-event-for-non-primary-button-click/1527">discourse</a>.
+      </p>
+    </section>
+
+    <section id='intro' class='informative'>
+      <h2>Introduction</h2>
+      <p>
+        This document mainly contains the new APIs that may not make it to Pointer Event Level 2 spec. So we iterate over these new APIs to make them ready for the next version.
```

Comment:
How about:
"This document reflects incubation for APIs which are intended to be part of a future version of the Pointer Events specification.  They are not yet broadly supported by implementations.".